website/src/pages/tutorial/basic/07-connecting-server-and-database.mdx

```diff
@@ -116,13 +116,15 @@ In `src/schema.ts`, update the `resolvers` object and change imports to look as
 
 ```ts filename="src/schema.ts"
 import { createSchema } from 'graphql-yoga'
-import { type Link } from '@prisma/client'
+import type { Link } from '@prisma/client'
 import type { GraphQLContext } from './context'
 
 const resolvers = {
   Query: {
     info: () => `This is the API of a Hackernews Clone`,
-    feed: (parent: unknown, args: {}, context: GraphQLContext) => context.prisma.link.findMany()
+    feed: async (parent: unknown, args: {}, context: GraphQLContext) => {
+      return context.prisma.link.findMany()
+    }
```

Comment:
This is updated to match the implementation mentioned in the `Understanding the feed Resolver` on the same page